File: QID-942-MVAdenbank2/MVAdenbank2.R

# clear variables and close windows
rm(list = ls(all = TRUE))
graphics.off()

# Observation: Density estimates are different because R uses a 'gaussian' kernel as default, whereas Xplore uses a Quartic Kernel.

# load data
x = read.table("bank2.dat")

x4 = x[1:100, 4]
x5 = x[1:100, 5]

f4 = density(x4)
f5 = density(x5)

# plot
par(mfrow = c(1, 2))
plot(f4, type = "l", lwd = 3, xlab = "Lower Inner Frame (X4)", ylab = "Density", main = "Swiss bank notes", 
    cex.lab = 1.2, cex.axis = 1.2, cex.main = 1.8)
plot(f5, type = "l", lwd = 3, xlab = "Upper Inner Frame (X5)", ylab = "Density", main = "Swiss bank notes", 
    cex.lab = 1.2, cex.axis = 1.2, cex.main = 1.8)
	

File: QID-942-MVAdenbank2/MVAdenbank2.m
%% clear loaded variables and close windows
clear all
close all
clc

%% load data
x  = load('bank2.dat')
x4 = x(1:100,4);
x5 = x(1:100,5);

[f1,xi1] = ksdensity(x4);
[f2,xi2] = ksdensity(x5);

%% plot
hold on
subplot(1,2,1)
plot(xi1,f1,'k','linewidth',2)
title('Swiss Bank Notes')
xlabel('Lower Inner frame (x4)')
ylabel('Density')
subplot(1,2,2)

plot(xi2,f2,'k','linewidth',2)
title('Swiss Bank Notes')
xlabel('Upper Inner Frame (X5)')
ylabel('Density')
hold off
